lib/puppet/defaults.rb

```diff
@@ -1570,7 +1570,10 @@ def self.default_diffargs
     :pluginsync => {
       :default    => true,
       :type       => :boolean,
-      :desc       => "Whether plugins should be synced with the central server.",
+      :desc       => "Whether plugins should be synced with the central server. This setting is
```

Comment:
@MikaelSmith @whopper legit. So, if I remember correctly, pluginsync used to be enabled by default, this changes that?